clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2336,6 +2336,8 @@ ParseResult cir::FuncOp::parse(OpAsmParser &parser, OperationState &state) {
   if (parser.parseOptionalKeyword(noProtoNameAttr).succeeded())
     state.addAttribute(noProtoNameAttr, parser.getBuilder().getUnitAttr());
 
+  // TODO: Missing comdat
```

Comment:
Good opportunity to add a `assert(!MissingFeatures::setComdat());` here

Reply:
I am unsure, because it is missing in the parser, not in the pretty-printer, so to have the assert to trip I would basically have to implement the missing parser.
This is simple but not on my critical path compared to have the LSP server working. :smiley:

Reply:
I think you got the wrong impression of this assert, it will only fire when someone changes the definition of `setComdat` to return `true`. This is more of a missing feature tracker than can later fire in all missing places once the definition is tweaked to return `true`. It's how we delay some of the features that would block everything if we were asserting for real.

Reply:
Oh, I see. This is actually a time bomb that will explode if we enable the `comdat` official support without implementing this at that place.